### PR TITLE
update css

### PR DESCRIPTION
**Pull request by michaelbabyn**
Addressing [this issue](https://github.com/plotly/dash-on-premise-sample-app/issues/20), I modified the app so that it uses the assets folder.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -14,6 +15,9 @@
     # Serve any files that are available in the `static` folder
     static_folder='static'
```
Comment (T4rk1n) [inline app.py]: :zap: If we use the assets folder, there is no need for the static folder anymore.

File: app.py [hunk 2]
```diff
@@ -14,6 +15,9 @@
...
 )
+
+app.config.assets_external_path = os.path.join('/',os.environ.get('DASH_APP_NAME', ''),'assets/')
```
Comment (T4rk1n) [inline app.py]: :zap: The `assets_external_path` is an url, not a file path, `os.path.join` will give the wrong slashes on windows. It is used to serve the assets externally on a cdn and the folder structure must match the local assets folder. You don't have to set this.

File: requirements.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-dash==0.21.1
+dash==0.22.0
```
Comment (T4rk1n) [inline requirements.txt]: Maybe put the latest, 0.26.0.

File: app.py (hunk 2 above)
Comment (michaelbabyn) [reply app.py]: Yeah, I figured it shouldn't be necessary but otherwise I was getting a 404 error when I deployed to DDS.

**Pull request merged**